### DISCUSSION TITLE
was stole tokens?

### DISCUSSION THREAD

**Discussion by Roman00000000** [Chia-Network/chia-blockchain#13416] [Support]
hello, I lost tokens from my wallet, what should I do???
and one more thing, rewrote the nucle app, wanted to go to the wallet, received a notification that my 12-word passphrase is not correct ...... my wallet was hacked, what should I do ?

**Comment by Iamalexoper** [comment]
The same problem! Nucle team wrote in the discord 10.09 : "We’re moving servers to our own server room to save money
So we will be out intermittently until that is over", but after that they are silent and the wallet does not work

**Comment by Roman00000000** [comment]
Ср, 14 сент. 2022 г. в 08:43, Iamalexoper ***@***.***>:

> The same problem! Nucle team wrote in the discord 10.09 : "We’re moving
> servers to our own server room to save money
> So we will be out intermittently until that is over", but after that they
> are silent and the wallet does not work
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Chia-Network/chia-blockchain/discussions/13416#discussioncomment-3641223>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3ABQQHOONILEZVY7YY3S53V6FQX7ANCNFSM6AAAAAAQJMFUMM>
> .
> You are receiving this because you authored the thread.Message ID:
> <Chia-Network/chia-blockchain/repo-discussions/13416/comments/3641223@
> github.com>
>
thank you, you can keep me informed, I will be very grateful

**Comment by shaneo257** [comment]
i would use a different wallet like goby or pawket, you can use the same 12 words from the nucle wallet and view your funds

**Reply by Iamalexoper** [reply]
thanks a lot, it worked!

**Comment by Roman00000000** [comment]
Helo
i don't see my tokens in nucle wallet , do you have some problem with you
wallet?

Чт, 15 сент. 2022 г. в 08:41, Iamalexoper ***@***.***>:
